xml/obs_admin.xml

```diff
@@ -9,15 +9,17 @@
 <chapter id="cha.obs.admin">
  <title>Administration</title>
  <para>
-  This chapter describes the components of an OBS installation and the typical
+  This chapter describes the components of an OBS installation
+   and the typical
   administration tasks for an OBS administrator.
  </para>
  <para>
   This chapter is not intended to describe special installation hints for a
-  certain OBS version, please refer the README.md file of the package for that.
```

Comment:
The README.md file was here on purpose, since it should be a file which is maintain inside of the specific git branch and not external without any connection to the branch.